### PR TITLE
Added about page with link from table view footer on issues screen

### PR DESCRIPTION
**Pull request by alexargo**
This is my quick and dirty solution for #7 since it seems like there's not much left for 1.0. This could be definitely be made native and much nicer in the future, but the basics are there. Slapped some html in a web view - Basically took what was available on the 5calls website and added iOS specific thinks to it (specific contributors, open sources libraries, etc.)

### CODE REVIEW

File: FiveCalls/FiveCalls/Info.plist [hunk 1]
```diff
@@ -2,11 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>NSAppTransportSecurity</key>
```
Comment (mccarron) [inline FiveCalls/FiveCalls/Info.plist]: Oh Xcode... why do you taunt us so??

File: FiveCalls/FiveCalls/AboutViewController.swift [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    @IBAction func close(_ sender: UIBarButtonItem) {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    override func viewDidLoad() {
+        if let asset = NSDataAsset(name: "AboutHtml") {
+            let htmlString = String(data: asset.data, encoding: .utf8)!
+            print(htmlString)
```
Comment (mccarron) [inline FiveCalls/FiveCalls/AboutViewController.swift]: We can probably remove this.

File: FiveCalls/FiveCalls/AboutViewController.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+class AboutViewController : UIViewController, UIWebViewDelegate {
+    
+    @IBOutlet weak var contentView: UIWebView!
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+
+    @IBAction func close(_ sender: UIBarButtonItem) {
+        self.dismiss(animated: true, completion: nil)
```
Comment (mccarron) [inline FiveCalls/FiveCalls/AboutViewController.swift]: Remove redundant `self` here.

File: FiveCalls/FiveCalls/AboutViewController.swift [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    func webView(_ webView: UIWebView, shouldStartLoadWith request: URLRequest, navigationType: UIWebViewNavigationType) -> Bool {
+        switch navigationType {
+        case .linkClicked:
+            // Open links in Safari
+            guard let url = request.url else { return true }
+            
+            if #available(iOS 10.0, *) {
+                UIApplication.shared.open(url, options: [:], completionHandler: nil)
+            } else {
+                // openURL(_:) is deprecated in iOS 10+.
```
Comment (mccarron) [inline FiveCalls/FiveCalls/AboutViewController.swift]: Oh forgot about this... my calling of `openURL:completion` call of this earlier is iOS 10 only. I'll have to fix that.

**Comment by mccarron** [comment]
Alternatively we can try to mimic some of these options from the web's footer on our footer.

screen shot 2017-02-06 at 9 02 09 am

**Comment by subdigital** [comment]
I'm making some modifications to this right now...

simulator screen shot feb 6 2017 10 23 06 am

Definitely need more things here, but I don't want to call it "Settings" :/

**Comment by subdigital** [comment]
Maybe "About" brings up a menu, which includes these rows:

- Why Callings Works
- Who are we?
- Help / Feedback
- Rate App
- Open Source
- Follow on Twitter
- Share app

**Comment by subdigital** [comment]
@alexargo I'm going to close this one in favor of #34, which includes these changes.

**Pull request closed without merging**